Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxOMSAutomationWorker.py

```diff
@@ -251,6 +251,12 @@ def test_is_any_1_4_process_running(self):
             '']
         self.assertFalse(nxOMSAutomationWorker.is_any_1_4_process_running(processes, self.workspace_id))
 
+    def test_is_certificate_valid(self):
+        self.assertTrue(nxOMSAutomationWorker.is_certificate_valid(nxOMSAutomationWorker.AUTO_REGISTERED_WORKER_CONF_PATH, os.path.join(self.dummyFileLocation, "dummy_match.crt")))
+        self.assertFalse(nxOMSAutomationWorker.is_certificate_valid(nxOMSAutomationWorker.AUTO_REGISTERED_WORKER_CONF_PATH, os.path.join(self.dummyFileLocation, "dummy_notMatch.crt")))
```

Comment:
Added the comments to the definition of the function nxOMSAutomationWorker.is_certificate_valid